File: ch2/test_opencv.cpp
/*****************************************************************************
 * Author : Ram
 * Date : 14/May/2018
 * Email : ramkalath@gmail.com
 * Breif Description : Testing OpenCV installation
 * Detailed Description : to compile type 'make' and to run './test_opencv'
 *****************************************************************************/

#include <iostream>
#include <opencv2/core/core.hpp>
#include <opencv2/highgui/highgui.hpp>

int main()
{
	cv::Mat img;
	img = cv::imread("test.jpg");
	cv::imshow("my first opencv window", img);
	cv::waitKey();
	return 0;
}

File: ch2/test_opencv.py
# *****************************************************************************
# * Filename : test_opencv.py
# * Date : 14/May/2018
# * Author : Ram
# * Email : ramkalath@gmail.com
# * Breif Description : Tests if opencv works properly using python
# * Detailed Description : to run the code type "python test_opencv.py"
# *****************************************************************************

import cv2
import numpy as np

image = cv2.imread("test.jpg")
cv2.imshow("my first opencv window", image)
cv2.waitKey()
